performance-timeline/resources/postmessage-entry.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<script>
+addEventListener("load", () => {
+  const entry = performance.getEntriesByType("navigation")[0];
+  try {
+    window.top.postMessage(entry, "*");
```

Comment:
If it does not throw then this will timeout. Would it make sense to add another postMessage below this one saying something like "FAIL - did not throw an error"? (I'm legitimately asking, I imagine a try will not execute further if it reaches an exception.

Reply:
Good call! Added.